packages/plans-grid-next/src/types.ts

```diff
@@ -138,37 +133,32 @@ export interface ComparisonGridProps extends CommonGridProps {
 	selectedPlan?: string;
 }
 
+export type UseActionCallbackParams = {
+	planSlug: PlanSlug;
+	// TODO: Explore if we can remove cartItemForPlan
+	cartItemForPlan?: { product_slug: string } | null;
+	selectedStorageAddOn?: AddOns.AddOnMeta | null;
+};
+
+export type UseActionCallback = ( params: UseActionCallbackParams ) => () => void;
```

Comment:
Curious - I'd have expected something like the following:

```
export type UseActionCallback = ( { planSlug, cartItemForPlan, selectedStorageAddOn }: UseActionCallbackParams ) => () => void;
```

so to not use a named object for the props

Reply:
Oh this didn't even cross my mind. I completely agree with you here. I think I had been too close to the code for too long 🤔 

Lemme make the change.

Reply:
https://github.com/Automattic/wp-calypso/pull/88694/commits/f2f1bbc98dfa1f890fbff24961058982d3495d40